jerry-core/lit/lit-magic-strings.inc.h

```diff
@@ -27,6 +27,7 @@ LIT_MAGIC_STRING_FIRST_STRING_WITH_SIZE (1, LIT_MAGIC_STRING_NEW_LINE_CHAR)
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_NEW_LINE_CHAR, "\n")
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_SPACE_CHAR, " ")
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_DOUBLE_QUOTE_CHAR,"\"")
+LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_SINGLE_QUOTE_CHAR,"'")
```

Comment:
Errors are usually short living objects. Magic strings are designed for property names which likely present for longer times. But having more magic strings does not hurt either.

Ok, we need a system which allows conditional magic string definition. There is no point adding error message strings when they never used. It would just increase binary size.